Lib/test/support/__init__.py

```diff
@@ -2728,7 +2727,6 @@ def iter_builtin_types():
     # Fall back to making a best-effort guess.
     if hasattr(object, '__flags__'):
         # Look for any type object with the Py_TPFLAGS_STATIC_BUILTIN flag set.
-        import datetime
```

Comment:
This import has to stay. It deserves a `# noqa: F401`.

@encukou: Can you confirm?